src/apb/engine.py

```diff
@@ -992,9 +992,9 @@ def cmdrun_push(**kwargs):
 
 def cmdrun_remove(**kwargs):
     if kwargs["all"]:
-        route = "/apb/spec"
+        route = "/ansible-service-broker/v2/apb"
```

Comment:
should we be making allowances for the downstream route (`openshift-ansible-service-broker`) the same way https://github.com/ansibleplaybookbundle/ansible-playbook-bundle/pull/183/files does?  It seems like perhaps we need a `find_asb_route` function so that these commands don't fail on downstream broker.